tests/conftest.py

```diff
@@ -54,7 +55,13 @@ def create_one_fake_venv(
 
         # Create fake packages
         major, minor = py_version
-        site_dir = venv_dir / f"lib/python{major}.{minor}/site-packages"
+
+        def _env_site_packages():
+            if platform.system() == "Windows":
+                return venv_dir / "Lib" / "site-packages"
+            return venv_dir / "lib" / f"python{major}.{minor}" / "site-packages"
+
+        site_dir = _env_site_packages()
```

Comment:
Isn't this easier?:

```suggestion
        if sys.platform.startswiith("win"):
            site_dir = venv_dir / "Lib" / "site-packages"
        else:
            major, minor = py_version
            site_dir = venv_dir / "lib" / f"python{major}.{minor}" / "site-packages"
```

Reply:
We can use `site_packages` function defined in `utils` for that.